appinfo/info.xml

```diff
@@ -20,7 +20,7 @@ A user logs into ownCloud with their LDAP or AD credentials, and is granted acce
 	<screenshot>https://raw.githubusercontent.com/owncloud/screenshots/0527af6b981603f3410982c2ed70051b7cf991c3/user_ldap/user_ldap.png</screenshot>
 	<dependencies>
 		<lib>ldap</lib>
-		<owncloud min-version="10.0" max-version="10.0" />
+		<owncloud min-version="10.0.1" max-version="10.0.99" />
```

Comment:
I don't know how it will be interpreted when ownCloud tries to install it. From my undestanding, 10.0.1 > 10.0 so the min version will be higher than the max version, at least in ownCloud.
I haven't tested that, but feel free to change it.